modules/delegates.js

```diff
@@ -1077,6 +1078,58 @@ Delegates.prototype.shared = {
 			return setImmediate(cb, null, delegates);
 		});
 	},
+
+	/**
+	 *
+	 * @param {Object} filters - Filters applied to results
+	 * @param {string} filters.address - Address of the delegate
+	 * @param {string} filters.start - Start time to aggregate
+	 * @param {string} filters.end - End time to aggregate
+	 * @params {function} cb - Callback function
+	 * @param {SetImmediateCallback} cb
+	 */
+	getForgingStatistics(filters, cb) {
+		// If need to aggregate all data then just fetch from the account
+		if (!filters.start && !filters.end) {
+			modules.delegates.getDelegates(
+				{ address: filters.address },
+				['rewards', 'fees', 'producedBlocks'],
+				(err, data) => {
+					if (err) {
+						return setImmediate(cb, err);
+					}
+
+					if (!data) {
+						return setImmediate(cb, 'Account not found');
```

Comment:
In which cases `data` will be empty? Maybe there is an account, but not a delegate?